sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -682,6 +682,14 @@ object functions {
    * Generate a random column with i.i.d. samples from U[0.0, 1.0].
    *
    * @group normal_funcs
+   * @since 1.5.0
+   */
+  def rand(seed: Int): Column = Rand(Int.int2long(seed))
```

Comment:
When I tried to mark them as `private[sql]`, I couldn't invoke them from SparkR.